### ISSUE TITLE
just a question

### ISSUE THREAD

**Issue by makabress** [phrutis/WifSolverCuda#19]
Hi,
thank you for all this work

i want add one gpu rtx 3060 on wifsolvercuda and two rtx 2060s in wif500
i want ask if this part of wif is correct 5bCRZhiS5sEGMpmcRZdPAhmWLRfMmutGmPHtjVob
and do i have to join the group for wif500 to ask for a range or i can take any one in the table list t

thank you

**Comment by phrutis** [comment]
No, this is the main mistake of many who are looking for separately.
In the uncompressed key, one letter less, many peck at it. The key is compressed.

**Comment by makabress** [comment]
ok so this is not the correct second part of the 500btc wif

**Comment by phrutis** [comment]
I'm assuming you have one letter wrong.
I hope you will not look for an uncompressed key

**Comment by makabress** [comment]
no the first transaction of the address is 2017 so i don't think it's uncompressed

**Comment by phrutis** [comment]
I agree with you, all child (already empty) addresses are compressed

**Comment by makabress** [comment]
ok thank you

**Comment by phrutis** [comment]
If you want to look for more on the processor, there is Fialka M-125

**Comment by makabress** [comment]
ok will try it too

**Comment by makabress** [comment]
First Start 3333 and 3334 range in wif500 see you in a week for finish
And i can you help to make the 40 character work for compressed, replaced many letters and stil work only for uncompressed i get no result with starting L and K
